src/configuration/evars.md

```diff
@@ -22,24 +22,50 @@ The `el` command to get help on all the evaluable configuration variables of riz
 
 The Visual mode has an eval browser that is accessible through the `Vbe` command.
 
-### asm.arch
+## Assembly Configuration
+
+### asm.arch: `string`
 
 Defines the target CPU architecture used for disassembling (`pd`, `pD` commands) and code analysis (`a` command). You can find the list of possible values by looking at the result of `e asm.arch=?` or `rz-asm -L`.
 It is quite simple to add new architectures for disassembling and analyzing code. There is an interface for that. For x86, it is used to attach a number of third-party disassembler engines, including GNU binutils, Udis86 and a few handmade ones.
 
-### asm.cpu
+### asm.bits: `int`
+
+Determines width in bits of registers for the current architecture. Supported values: 8, 16, 32, 64. Note that not all target architectures support all combinations for asm.bits.
+
+### asm.bytes: `bool`
+
+A boolean value used to show or hide displaying of raw bytes of instructions.
+
+### asm.cpu: `string`
 
 You can use this configuration variable to define the CPU type. For example, if you had picked the architecture as AVR, you can choose your CPU type (ATmega1281, ATmega2561, etc) using `asm.cpu`.
 
-### asm.bits
+### asm.flags: `bool`
 
-Determines width in bits of registers for the current architecture. Supported values: 8, 16, 32, 64. Note that not all target architectures support all combinations for asm.bits.
+If defined to "true", disassembler view will have flags column.
 
-### asm.syntax
+### asm.lines.call: `bool`
 
-Changes syntax flavor for disassembler between Intel and AT&T. At the moment, this setting affects Udis86 disassembler for Intel 32/Intel 64 targets only. Supported values are `intel` and `att`.
+If set to "true", draw lines at the left of the disassemble output (`pd`, `pD` commands) to graphically represent control flow changes (jumps and calls) that are targeted inside current block. Also, see `asm.lines.out`.
+
+### asm.lines.out: `bool`
+
+When defined as "true", the disassembly view will also draw control flow lines that go outside of the block.
+
+### asm.linestyle: `bool`
```

Comment:
the name for this config is no clear and does not match the description. Probably need to improve it in different PR for Rizin

Reply:
Will do it on another PR after discussing and after looking into it.